src/tail_csv.c

```diff
@@ -277,6 +278,28 @@ static int tcsv_config_get_index(oconfig_item_t *ci, ssize_t *ret_index) {
   return 0;
 }
 
+static int tcsv_config_get_separator(oconfig_item_t *ci, char *ret_char) {
+  size_t len_opt;
+
+  if ((ci->values_num != 1) || (ci->values[0].type != OCONFIG_TYPE_STRING)) {
+    WARNING("tail_csv plugin: The \"%s\" config option needs exactly one "
+            "string argument.",
+            ci->key);
+    return -1;
+  }
+
+  len_opt = strlen(ci->values[0].value.string);
```

Comment:
Policy ask us to declare variables at first use.

Reply:
Thanks for looking at this @rpv-tomsk , the variable has been declared at L282 unless I am missing something here?

Reply:
https://collectd.org/review-comments#initialize-variables
https://collectd.org/review-comments#define-variables-on-first-use